webpack-stats.json

```diff
@@ -0,0 +1 @@
+{"status":"done","chunks":{"main":[{"name":"main-6b65df00330569239f5d.js","path":"/betterself/assets/bundles/main-6b65df00330569239f5d.js"}]}}
```

Comment:
This probably then auto fails on a non-local box ... how do you get a script to rewrite?